letsencrypt/client/crypto_util.py

```diff
@@ -45,20 +45,20 @@ def create_sig(msg, key_str, nonce=None, nonce_len=CONFIG.NONCE_SIZE):
     hashed = Crypto.Hash.SHA256.new(msg_with_nonce)
     signature = Crypto.Signature.PKCS1_v1_5.new(key).sign(hashed)
 
-    logging.debug('%s signed as %s', msg_with_nonce, signature)
+    logging.debug("%s signed as %s", msg_with_nonce, signature)
 
     n_bytes = binascii.unhexlify(_leading_zeros(hex(key.n)[2:].rstrip("L")))
     e_bytes = binascii.unhexlify(_leading_zeros(hex(key.e)[2:].rstrip("L")))
 
     return {
-        "nonce": le_util.jose_b64encode(nonce),
-        "alg": "RS256",
-        "jwk": {
-            "kty": "RSA",
-            "n": le_util.jose_b64encode(n_bytes),
-            "e": le_util.jose_b64encode(e_bytes),
+        'nonce': le_util.jose_b64encode(nonce),
+        'alg': "RS256",
+        'jwk': {
+            'kty': "RSA",
```

Comment:
"Be consistent with your choice of string quote character within a file. " (https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Strings#Strings)